sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -171,6 +175,43 @@ internal class RadarApiClient(
         )
     }
 
+    // api handler for /track/replay, just takes in a list of replays to send to API
+    internal fun replay(replays: List<RadarReplay>, callback: RadarReplayApiCallback?) {
+        val publishableKey = RadarSettings.getPublishableKey(context)
+        if (publishableKey == null) {
+            callback?.onComplete(RadarStatus.ERROR_PUBLISHABLE_KEY)
+
+            return
+        }
+
+        val params = JSONObject()
+        val replaysList = mutableListOf<JSONObject>()
+        for (replay in replays) {
+            replaysList.add(replay.replayParams)
+        }
+        params.putOpt("replays", JSONArray(replaysList))
+
+        val path = "v1/track/replay"
+        apiHelper.request(
+            context = context,
+            method = "POST",
+            path = path,
+            headers = headers(publishableKey),
+            params = params,
+            sleep = false,
+            callback = object : RadarApiHelper.RadarApiCallback {
+                override fun onComplete(status: RadarStatus, res: JSONObject?) {
+                     if (status != RadarStatus.SUCCESS) {
+                        Radar.sendError(status)
+                     }
+
+                    callback?.onComplete(status, res)
+                }
+            },
+            extendedTimeout = true,
```

Comment:
Let's keep this as `true` to match current behavior, but given our quick 200 loop with `/track/replay` from the fanout, we probably can soon change this to `false`.